.github/workflows/release-image.yaml

```diff
@@ -0,0 +1,38 @@
+name: Generate Release Image
+
+on: [pull_request]
```

Comment:
I'm having second thoughts about whether having this as a separate workflow like this is preferable. It adds an extra line in the list of checks for all opened PR, though this one would be skipped unless it was made by lit-robot.

I think in our discussion offline, we were considering if we want to edit the PR description and re-run it, though I think this wouldn't run in that case because if we're the one re-triggering, is the `github.actor` value still lit-robot?

Reply:
The primary reason I moved it to a separate workflow is so the image artifact is local and attached to the release PR. Otherwise the image artifact ends up on the "released" commit, instead of the pull request. This makes the DX of finding the release image a little harder.

Another alternative is generating the image in the release action, and then adding a "download artifact" step somewhere on the "Version Packages" PR. However that would probably still need an additional action.

Reply:
I see.. The trigger is run on the latest commit in main that we approve. I suppose it might be possible to have it run there, but add a link to the artifact as a comment to the Version Packages PR? Though apparently there isn't an easy way to do that https://github.com/actions/upload-artifact/issues/50. We'd likely need another step/script that uses github api to fetch the artifact id to create a link to it since the artifact upload workflow doesn't provide that, and another api call to post the comment to the correct PR.

It shouldn't be a huge deal but I'd prefer if we don't have to see the "Generate Release Image" workflow on every single PR. But maybe the effort to achieve that is not worth it when functionally this works.

Reply:
This comment thread was resolved in https://github.com/lit/lit/pull/3176 by using the Github Actions `paths` filter.  Thank you!